Sources/Paywalls/Events/PaywallEvent.swift

```diff
@@ -16,6 +16,9 @@ import Foundation
 /// An event to be sent by the `RevenueCatUI` SDK.
 public enum PaywallEvent {
 
+    /// An identifier that represents a paywall event.
+    public typealias EventID = UUID
```

Comment:
Maybe
```suggestion
    public typealias ID = UUID
```
So this becomes `PaywallEvent.ID`?